src/ProxyClient/Invalidation/ClearCacheCapable.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\ProxyClient\Invalidation;
+
+use FOS\HttpCache\ProxyClient\ProxyClient;
+
+/**
+ * An HTTP cache that supports clearing all of its
+ * cache completely.
```

Comment:
s/its cache completely/its cache entries/

can we add a second paragraph? saying that this is here to allow a more efficient operation than banning everything, and might work with caches that do not support banning.